src/Visualization/Python/ReadInputFile.py

```diff
@@ -44,3 +41,25 @@ def find_event(event_name: str, input_file: dict) -> dict:
                 if event_name in event:
                     return event[event_name]
     return None
+
+
+def find_phase_change(phase_change_name: str, input_file: dict) -> dict:
+    """Find a particular phase change in the "PhaseChangeAndTriggers"
+
+    Arguments:
+      phase_change_name: The name of a phase change like
+        "CheckpointAndExitAfterWallclock".
+      input_file: The input file read in as a dictionary.
+
+    Returns: The phase change as a dictionary, or None if the phase change
+      wasn't found.
+    """
+    if not "PhaseChangeAndTriggers" in input_file:
+        return None
+    for trigger_and_phase_changes in input_file["PhaseChangeAndTriggers"]:
+        for phase_change in trigger_and_phase_changes["PhaseChanges"]:
+            if isinstance(phase_change, str):
```

Comment:
The phase change `CheckpointAndExitAfterWallclock` is a string though and we want that returned if it's there

Reply:
Oh I just used it wrong, the empty dict that's returned here evaluates to False in an if condition, so I changed it to `is not None`

Reply:
Ok now it works!